components/Card.tsx

```diff
@@ -0,0 +1,91 @@
+import { getFormattedDate, getTimeAgo } from "@/lib/date";
+import noImage from "@/public/noImg.svg";
+import kebab from "@/public/kebab.svg";
+import emptyStar from "@/public/emptyStar.svg";
+import styles from "@/styles/Card.module.css";
+import React, { SyntheticEvent, useEffect, useState } from "react";
+import { UserContext } from "./FolderDetail";
+import { UserData } from "../pages/Folder";
+import Link from "next/link";
+import Image from "next/image";
+
+interface CardProps {
+  data: {
+    id: number;
+    created_at: string;
+    updated_at: null;
```

Comment:
아마도 이렇지 않을까요? 업데이트 일자가 값이 계속 null일리는 없으니까요!
```suggestion
    updated_at: string | null;
```

Reply:
api 문서에는 전부 null이어서 이렇게 했습니다.

Reply:
네~ 이해했습니다!
실제로 일을 하다보면 동료 개발자도 충분히 실수할 수 있기 때문에 내가 느끼기에 잘못된 부분이라고 생각하면 확인하는 과정이 필요하다고 생각해주시면 좋을 것 같아요~ 그리고 이 부분은 api 문서가 잘못된걸로 보이네요!

Reply:
넵 알겠습니다 !